core/arch/arm/mm/tee_mmu.c

```diff
@@ -263,7 +263,7 @@ TEE_Result tee_mmu_init(struct tee_ta_ctx *ctx)
 TEE_Result tee_mmu_map(struct tee_ta_ctx *ctx, struct tee_ta_param *param)
 {
 	TEE_Result res = TEE_SUCCESS;
-	paddr_t pa;
+	paddr_t pa = NULL;
```

Comment:
Wouldn't it be better to change the prototype of `core_va2pa()` to `paddr_t core_va2pa(void *va);`? It seems to me there is no need for an output parameter *and* a return value, unless a physical address of 0 is considered valid -- which currently cannot be true since we don't build with `-fno-delete-null-pointer-checks`.
Any opinions on this?